src/pages/home/sections/episodes/episode-small.js

```diff
@@ -83,9 +83,9 @@ EpisodeSmall.styles = StyleSheet.create({
     },
   },
   guestsAnchor: {
-    color: 'gray',
+    color: '#634806',
```

Comment:
Someone fixed this one, but the other two changes are needed! Could you update your pull request to include only those changes? Thanks!